instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -132,6 +141,9 @@ func (c *config) handleRPC(ctx context.Context, rs stats.RPCStats, isServer bool
 	if gctx != nil {
 		metricAttrs = make([]attribute.KeyValue, 0, len(gctx.metricAttrs)+1)
 		metricAttrs = append(metricAttrs, gctx.metricAttrs...)
+		if !gctx.record {
```

Comment:
How about moving it 2 lines above to reduce the amount of heap allocations when the call is filtered?

Reply:
I totally agree with you, I changed the location.
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5196/commits/24f7819a914cc754de75de89578588c6f71f84cb